app/admin/legislations.rb

```diff
@@ -24,10 +25,17 @@
     column :framework
     column :location
     column :document_types
+    tag_column :visibility_status
```

Comment:
I think we could add active admin scopes on that column like "All | Draft | Pending | Published"